### ISSUE TITLE
[Protocol] Call `CreateClaim` and `SubmitProof` from the `relayminer`

### ISSUE THREAD

**Issue by Olshansk** [pokt-network/poktroll#143]
## Objective

Integrate the SmartClient with the claim & proof lifecycle

## Origin Document

- #140
- #141
- #142

## Goals

- Unblock iterative development of the Applicaiton / Gateway / SDK / SmartClient on the Claim & Proof Lifecycle

## Deliverables

- [ ] A single PR that updates the Client (Application/Gateway) logic to correctly submit & verify the `CreateClaim` and `SubmitProof` messages

## Non-goals / Non-deliverables

- Probabilistically spreading out the frequency of Claim & Proof messages (leave TODOs or create another ticket)
- Dependencies for on-chain governance params related to submitting Claims & Proofs (leave TODOs and address in #142)

## General deliverables

- [ ] **Comments**: Add/update TODOs and comments alongside the source code so it is easier to follow.
- [ ] **Testing**: Add new tests (unit and/or E2E) to the test suite.
- [ ] **Makefile**: Add new targets to the Makefile to make the new functionality easier to use.
- [ ] **Documentation**: Update architectural or development READMEs; use [mermaid](https://mermaid-js.github.io/mermaid/) diagrams where appropriate.

---

**Creator**: @Olshansk 
**Co-Owners**: @bryanchriswhite  @red-0ne

**Comment by Olshansk** [comment]
@red-0ne Do you know what work/scope this ticket entails?

**Comment by red-0ne** [comment]
Claims & Proofs are already called from the relayminer. The missing parts are captured in the `Non-deliverables` and have their respective TODOs in the code.

**Comment by Olshansk** [comment]
@red-0ne Provided that:

1. TODOs are in the code
2. Remaining work is in non-deliverables

Why have we not closed this ticket out?

**Comment by red-0ne** [comment]
We delayed looking at it till after migration.

Closing now
